vl-proza-message.src.js

```diff
@@ -79,15 +79,13 @@ export class VlProzaMessage extends VlElement(HTMLElement) {
     }
 
     static _getMessage(domain, code) {
-        if (VlProzaMessagePreloader.isPreloaded(domain)) {
-            return VlProzaMessagePreloader.getMessage(domain, code);
-        } else {
+        return VlProzaMessagePreloader.getMessage(domain, code).catch(() => {
```

Comment:
Moet er ook geen console warn gelogd worden om de afnemers duidelijk te maken dat de preload niet werkt? Een fallback is goed, maar zonder melding aan de afnemer is misschien gevaarlijk.

Reply:
True, zal dat toevoegen.